app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    lst = command.split(' ')
+    if os.path.isfile(lst[1]):
+        if lst[2].find("/") > -1 and os.path.exists(lst[2]) is False:
```

Comment:
Use unpacking (ex. a, b = [1, 2]) and avoid accessing command items by index. 

`lst` is a bad name for the variable

Reply:
Уточните почему стоит избегать доступа к элементам команды по индексу?

Reply:
When the list is small (2-3 items) and list items are reused several times, it's simpler to assign those items to the variables and use variables instead of hardcoding list indexes
It looks more readable